src/utils/src/deviceinfo.h

```diff
@@ -27,6 +27,8 @@ class DeviceInfo : public QObject
     Q_OBJECT
     Q_DISABLE_COPY(DeviceInfo)
     Q_PROPERTY(bool hasRoundScreen READ hasRoundScreen CONSTANT)
+    Q_PROPERTY(double  edgeBoundary READ edgeBoundary CONSTANT)
```

Comment:
In this file you use double spaces before variable names for no apparent reason

Reply:
Alignment was the initial reason. But once I added edgeBoundary it all fell out of place :smile:

Reply:
Maybe change the name of this function to edgeSensitivity instead?

Reply:
Maybe something like BorderGestureWidth ?
This isn't really a sensitivity and this is only an edge for the BorderGestureArea item.

Reply:
oh, yeah even better!